gandalf.js

```diff
@@ -5,48 +5,51 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.